app/models/requests/IdentifierRequest.scala

```diff
@@ -16,7 +16,12 @@
 
 package models.requests
 
-import models.{SmallProducer,ReturnPeriod}
+import models.retrieved.RetrievedSubscription
+import models.{ReturnPeriod, SmallProducer}
 import play.api.mvc.{Request, WrappedRequest}
 
-case class IdentifierRequest[A] (request: Request[A], sdilEnrolment: String, smallProducerList :List[SmallProducer] = List.empty, returnPeriod: Option[ReturnPeriod] = None) extends WrappedRequest[A](request)
+case class IdentifierRequest[A] (request: Request[A],
+                                 sdilEnrolment: String,
+                                 subscription: RetrievedSubscription,
+                                 //smallProducerList :List[SmallProducer] = List.empty,
```

Comment:
Remove this line please, we can always refer back to commit history if we need something from the past.